bot/kodiak/evaluation.py

```diff
@@ -435,7 +435,13 @@ async def set_status(msg: str, markdown_content: Optional[str] = None) -> None:
         )
         return
 
-    merge_method = config.merge.method
+    if config.merge.method is not None:
+        merge_method = config.merge.method
+    elif len(valid_merge_methods) == 1:
+        merge_method = valid_merge_methods[0]
+    else:
+        merge_method = MergeMethod.merge
```

Comment:
this seems a little weird. If there is only one then taking the first makes sense, but here if there are multiple we don't choose any of them.

Instead we should pick the first one and maybe log something to sentry if there are multiple values